python/pyspark/pandas/window.py

```diff
@@ -706,11 +706,16 @@ def _apply_as_series_or_frame(self, func: Callable[[Column], Column]) -> FrameLi
         if groupby._agg_columns_selected:
             agg_columns = groupby._agg_columns
         else:
-            agg_columns = [
-                psdf._psser_for(label)
+            agg_column_labels = [
+                label
                 for label in psdf._internal.column_labels
                 if label not in groupby._column_labels_to_exlcude
```

Comment:
Can we add the column label to `column_labels_to_exlcude` (with some comments to explain why) at `DataFrameGroupBy` constructor?

Reply:
Got it, and seems like we should also rename `exlcude` to `exclude` ??

It's typo 😮